### PR TITLE
add new icon

### PR DESCRIPTION
**Pull request by a26620236**
# Purpose
add new icon 
image

# Changes

- a list of what have been done
- maybe some code change

# Risk

Usually none, if you have any please write it here.

# TODOs

- [ ] Describe what should be done outside of this PR
- [ ] Maybe in other PRs or some manual actions.

### CODE REVIEW

**Review by chenesan — APPROVED** [review]

**Review by YuCJ — APPROVED** [review]

File: packages/core/src/icons/components/Update.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      width="1em"
+      height="1em"
+      viewBox="0 0 18 18"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        d="M.477 9.148h1.181a.443.443 0 00.443-.444 6.579 6.579 0 011.368-4.036A6.6 6.6 0 018.75 2.066c1.469 0 2.862.472 4.029 1.365.386.296.734.627 1.04.99l-2.18-.386a.443.443 0 00-.513.36l-.205 1.162c-.043.24.118.47.359.513l5.232.924a.441.441 0 00.513-.36l.924-5.232a.443.443 0 00-.36-.513L16.429.684a.442.442 0 00-.513.359l-.389 2.202a8.747 8.747 0 00-1.49-1.455A8.621 8.621 0 008.752 0a8.653 8.653 0 00-6.924 3.412A8.629 8.629 0 00.034 8.706a.443.443 0 00.443.442zM17.836 8.982a.443.443 0 00-.313-.13h-1.18a.443.443 0 00-.443.444 6.58 6.58 0 01-1.369 4.036 6.6 6.6 0 01-5.282 2.602A6.574 6.574 0 015.22 14.57a6.68 6.68 0 01-1.04-.989l2.18.385a.442.442 0 00.513-.36l.205-1.162a.443.443 0 00-.359-.513l-5.232-.924a.443.443 0 00-.513.36L.05 16.597c-.042.24.119.47.36.513l1.162.205a.443.443 0 00.513-.36l.389-2.201c.436.54.937 1.031 1.49 1.455A8.62 8.62 0 009.248 18a8.653 8.653 0 006.924-3.412 8.63 8.63 0 001.794-5.294.443.443 0 00-.13-.312z"
+        fill="#231F20"
```
Comment (kyoyadmoon) [inline packages/core/src/icons/components/Update.js:15]: 這個 fill 是不是應該換成 currentColor

Comment (a26620236) [reply packages/core/src/icons/components/Update.js:15]: 沒錯～感謝提醒

**Review by zhusee2 — APPROVED** [review]

File: packages/core/src/icons/components/Refresh.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function SvgRefresh(props) {
+  return (
+    <svg
+      width="1em"
+      height="1em"
+      viewBox="0 0 18 18"
```
Comment (kyoyadmoon) [inline packages/core/src/icons/components/Refresh.js:8]: 看了一下 preview ，好像長的稍微大一點
之前的 viewBox 好像都是 32 要不要再確認一下尺寸？

**Review by kyoyadmoon — APPROVED** [review]
記得要用 conventional commit
https://www.conventionalcommits.org/en/v1.0.0/

**Review by benny0642 — APPROVED** [review]

**Pull request merged**